resources/js/requests/components/FilterMobile.vue

```diff
@@ -0,0 +1,281 @@
+          >
+            <i :class="selectedIconFilter"/>
+            <i class="fas fa-caret-down"/>
+          </button>
+          <div
+            class="dropdown-menu"
+            aria-labelledby="requestsDropdown"
+          >
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption(`(requester = 'admin')`, 'filter', 'fas fa-user')"
+            >
+              <i class="fas fa-user"/>
+              {{ $t('Requested by Me') }}
+              <i v-if="selectedIconFilter=== 'fas fa-user'" class="fas fa-check ml-auto text-success"/>
+            </a>
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption(`(participant = 'admin')`, 'filter', 'fas fa-users')"
+            >
+              <i class="fas fa-users"/>
+              {{ $t('With me as Participant') }}
+              <i v-if="selectedIconFilter === 'fas fa-users'" class="fas fa-check ml-auto text-success"/>
+            </a>
+          </div>
+        </div>
+        <div
+          class="dropdown"
+          v-if="showDropdowns && type === 'tasks'"
+        >
+          <button
+            id="tasksDropdown"
+            class="btn btn-secondary dropdown-toggle"
+            type="button"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
+          >
+            <i class="fas fa-list"/>
+            <i class="fas fa-caret-down"/>
+          </button>
+          <div
+            class="dropdown-menu"
+            aria-labelledby="tasksDropdown"
+          >
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption('By Due Date', 'orderBy')"
+            >
+              {{ $t('By Due Date') }}
+            </a>
+            <a
+              class="dropdown-item"
+              href="#"
+              @click="selectOption('By Creation Date', 'orderBy')"
+            >
+              {{ $t('By Creation Date') }}
+            </a>
+          </div>
+        </div>
+      </b-col>
+      <div class="d-flex align-items-end">
+        <button
+          class="btn btn-primary ml-2"
+          @click="toggleInput"
+        >
+          <i class="fas fa-search"/>
+        </button>
+        <input
+          v-if="showInput"
+          ref="input"
+          type="text"
+          class="form-control narrow-input"
+          v-model="searchCriteria"
+          placeholder="(fulltext LIKE '%someText%')"
+          @keyup.enter="performSearch"
+        />
+        <button
+          v-if="showInput"
+          class="btn btn-clear"
+          @click="clearSearch"
+        >
+        <i class="fas fa-times"/>
+    </button>
+      </div>
+    </b-row>
+  </b-container>
+</template>
+<script>
+export default {
+  props: {
+    type: String,
+  },
+  data() {
+    return {
+      searchCriteria: "",
+      selectedOptionStatus: "In Progress",
```

Comment:
Is this label translated somewere else?

Reply:
not necessary in this section is data to send an API